### ISSUE TITLE
{Freeway-Bug} [Voice Access> ActivityUpdate.json> Set Due Date]: Name is not defined for the 'Calendar date picker' edit field under 'Set Due Date' of 'Adaptive Cards.json'.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7516]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro steps:**
1. Turn on Voice access.
2. Open Adaptive Cards UWP Application.
3. Command: Show numbers
4. Select particular number for 'ActivityUpdateWithLabel.json'
5. Command: Move to 'Set due date'
6. Command: Click
7. Now, try to access the edit field of calendar date picker
8. Observe the issue.

**Actual Result:**
Name is not defined for the 'Calendar date picker' edit field under 'Set Due Date' of 'Adaptive Cards.json'.

**Expected Result:**
Proper name should be defined for the 'Calendar date picker' edit field under 'Set Due Date' of 'Adaptive Cards.json'.

**User Impact:**
Voice access and Screen reader user will have difficulty in interacting with the controls as they will not be able to know the purpose of the control.

**MAS reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

<img width="959" alt="MAS4 1 2 Name property of the calendar date picker under activityupdate is null" src="https://user-images.githubusercontent.com/104483005/173387802-da3cf22e-7421-4190-afbf-424b6d558b9e.png">
<img width="960" alt="MAS4 2 1 Name property is not defined for calendar date picker" src="https://user-images.githubusercontent.com/104483005/173410577-eaa3d3f7-82c6-4103-8ca9-7ebabea41f41.png">

### Card JSON

```JSON
NA
```

**Comment by vaishali1397** [comment]
Issue is repro'ing in below test Environment, hence reactivating the bug,
**Test Environment:**
OS Version: 22H2 (Build25272.1000)
Application: Adaptive card Visualizer

![image](https://user-images.githubusercontent.com/104483005/212269049-24dc861d-01b7-41ee-90a1-9f8d49b78dae.png)

**Comment by JeanRoca** [comment]
This issue has been fixed and will be included in our next release. I am closing this issue for now. Once new version is available, if the issue persists please open a new issue to track.

**Comment by vaishali1397** [comment]
Issue has been fixed in below test Environment, hence Adding A11yTTValidated tag.
Test Environment:
OS Version: 22H2 (Build 25314. 1010)
Application: Adaptive Card Visualizer (Version 3.2.2303.22003)

![image](https://user-images.githubusercontent.com/104483005/228222123-119f091f-36c6-4b52-ac5c-9a7e46bd5398.png)
